data/examples/shaders/filter/rounded-rect.effect

```diff
@@ -0,0 +1,85 @@
+#define IS_FILTER
+#include "../base.effect"
+
+//------------------------------------------------------------------------------
+// Uniforms
+//------------------------------------------------------------------------------
+
+uniform int _0_corner_radius<
+	string name = "Corner Radius";
+	string suffix = " px";
+	string field_type = "input";
+	int minimum = 0;
+	int maximum = 2000;
+	int step = 10;
+	int scale = 1;
+> = 40;
+
+uniform float4 _1_border<
+	string name = "Border Color";
+	string field_type = "slider";
+	float4 minimum = {0., 0., 0., 0.};
+	float4 maximum = {100., 100., 100., 100.};
+	float4 step = {0.01, 0.01, 0.01, 0.01};
+	float4 scale = {0.01, 0.01, 0.01, 0.01};
+> = {0., 0., 0., 100.};
+
+uniform int _2_border_thickness<
+	string name = "Border Thickness";
+	string suffix = " px";
+	string field_type = "input";
+	int minimum = 0;
+	int maximum = 300;
+	int step = 1;
+	int scale = 1;
+> = 5;
+
+
+//------------------------------------------------------------------------------
+// Technique: Combined Curvature
+//------------------------------------------------------------------------------
+
+float4 CombinedCurvature(VertexInformation vtx) : TARGET {
```

Comment:
**Optional:** Consider renaming the pixel shader to match the actual functionality.